operator/controllers/runtimetask_controller_test.go

```diff
@@ -0,0 +1,820 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	"k8s.io/kubeadm/operator/errors"
+
+	operatorv1 "k8s.io/kubeadm/operator/api/v1alpha1"
+)
+
+func TestRuntimeTaskReconciler_reconcilePhase(t *testing.T) {
+	tx := timePtr(metav1.Now())
+
+	type args struct {
+		task *operatorv1.RuntimeTask
+	}
+	type want struct {
+		task *operatorv1.RuntimeTask
+	}
+	tests := []struct {
+		name string
+		args args
+		want want
```

Comment:
as mentioned on the @SataQiu PR this should be, like so:
https://github.com/kubernetes/kubeadm/pull/1923/files#diff-1175c7d3cbb2ee3af06c7f1bbd008d60R34-R36

Reply:
[1] I don't see any value added in renaming args in input and want in output, so I don't think this should block the PR

TBH I have to say that the PR from @SataQiu deviated from the example provided, without discussing the point with the author of the issue